raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -178,6 +179,14 @@ def is_send_transfer_almost_equal(
 
 
 def filter_available_routes(transfers_pair, routes):
+    """This function makes sure we don't try routes that have already been tried
+       So in a setup like this, we want to make sure than node 2, having tried to
```

Comment:
1. s/than/that/ ? 
1. the space on the beginning is wrong, I'm not sure if it will typeset correctly with sphinx.
1. add a new line separating the short description from this?